stats/cloud/api.go

```diff
@@ -87,6 +91,51 @@ func (c *Client) PushMetric(referenceID string, samples []*Sample) error {
 	return err
 }
 
+func (c *Client) StartCloudTestRun(name string, arc *lib.Archive) (string, error) {
+	requestUrl := fmt.Sprintf("%s/archive-upload", c.baseURL)
+
+	buf := bytes.NewBuffer(nil)
```

Comment:
If you're not preinitialising the buffer with anything, `var buf bytes.Buffer` is better.

https://godoc.org/bytes#Buffer